### PR TITLE
Update Topics Page for pagination and more

### LINKED ISSUE #8571: Reinstate loading of additional resources in the topics page
## Observed behavior
Currently we display at most 4 items within a topic in the topics page - but have no option on how to show any more.

## Expected behavior
If there are 4 more results available locally, then it should immediately display them.
If not, it should load more from the backend and show a loading state until the request is completed.

## User-facing consequences
Confusion as only 4 items in a topic are ever shown!

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary
* Adds the ability to show more resources inline in the TopicsPage when they have been cut off
* Adds ability to navigate to a topic subpage to see all loaded resources
* Adds ability to load additional resources within the topic subpage
* Adds ability to load additional topics in the side bar
* Consolidates all folder selection logic in the topics page in the side bar
* Move breadcrumbs above the topic header
* Updates header to h1 (from h3)
* Adds an optional clearable state to KSelect
* Uses this for all KSelects used in categorical search to provide an easy way of deselecting.

## References
Fixes #8571 and some unfiled design updates

## Reviewer guidance
Do the interactions all work? Do the code changes make sense? Are we loading enough items at once?

1. Show more inline:
showmore

2. View all loaded
viewall

3. View more within a topic
viewmore

4. View more topics
viewmoretopics

5. Clearable dropdown for label selection
clearabledropdown

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [x] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by rtibbles** [comment]
Updates from review by @jtamiace

Better spacing of the link buttons beneath card grids:
Screenshot from 2021-11-04 15-41-06

Adding additional View more button to supplement folder side bar
moreviewmore

Showing more on first load of View all subpage
moreviewall

**Review by jtamiace — APPROVED** [review]
👍🏼 from a design POV

File: kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue [hunk 1]
```diff
@@ -135,18 +142,6 @@
...
-        key = 'ToUseWithTeachersAndPeers';
-      } else if (key === 'PAPER_PENCIL') {
-        key = 'ToUseWithPaperAndPencil';
-      } else if (key === 'INTERNET') {
-        key = 'NeedsInternet';
-      } else if (key === 'MATERIALS') {
-        key = 'NeedsMaterials';
-      } else if (key === 'FOR_BEGINNERS') {
-        key = 'ForBeginners';
-      }
```
Comment (marcellamaki) [inline kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue:149]: How are you able to delete this? 🤔

**Review by marcellamaki — COMMENTED** [review]
Overall, it seems to be working, and although I can't speak to the 'optimization' or performant side of things, I find the code clearly written and I could follow along (which is saying a lot!). My only question is on some of the deleted code (added a comment) -- otherwise I think it's okay to merge and if there are some other optimizations that could be added that I would not think of, probably can be a follow up issue

File: kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue (hunk 1 above)
Comment (rtibbles) [reply kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue:149]: For the benefit of future reviewers - this was code that had already been ported into the commonCoreStrings mixin, so is now redundant.

**Review by marcellamaki — APPROVED** [review]

**Comment by pcenov** [comment]
Tested and verified that all's implemented as described in the following [build ](https://buildkite.com/learningequality/kolibri-python-package/builds/4048) @radinamatic

**Pull request merged**